src/main/java/MockClass.java

```diff
@@ -1,8 +1,17 @@
 
 public class MockClass {
 
+    /**
+     * cioc
+     * @param seed
+     * @return
+     */
     public int doSomething(int seed) {
 
+        System.out.println("message");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tiolktrack.byznass.com:9000/coding_rules#rule_key=squid%3AS106)